libkbfs/dirty_bcache.go

```diff
@@ -67,44 +115,46 @@ type DirtyBlockCacheStandard struct {
 	// accepted. Used only for testing.
 	blockedChanForTesting chan<- int64
 
-	// The minimum (and initial) size of the sync buffer.
-	minSyncBufferSize int64
-	// The maximum size of the sync buffer.  Also used as the
+	// The minimum (and initial) capacity of the sync buffer.
+	minSyncBufCap int64
+	// The maximum capacity of the sync buffer.  Also used as the
 	// denominator when calculating backpressure, such that the closer
 	// we are to reaching the maximum size (over and above the current
 	// sync buffer), the more write requests will be delayed.
-	maxSyncBufferSize int64
+	maxSyncBufCap int64
 
 	shutdownLock sync.RWMutex
 	isShutdown   bool
 
-	lock               sync.RWMutex
-	cache              map[dirtyBlockID]Block
-	unsyncedDirtyBytes int64
-	syncingDirtyBytes  int64 // just for bookkeeping, not actually used
-	totalDirtyBytes    int64
-	syncBufferSize     int64
+	lock            sync.RWMutex
+	cache           map[dirtyBlockID]Block
+	syncBufBytes    int64
+	waitBufBytes    int64
+	syncBufferCap   int64
+	ignoreSyncBytes int64 // these bytes have "timed out"
+	syncStarted     time.Time
```

Comment:
@jzila I'm not 100% sure I follow.  Do you mean you want to print out `syncBufferBytes`, `waitBufBytes`, and `syncBufferCap` (any others) every time one of them change?  Or do you mean something else?

While working on this I tried having printouts in the `Updating...` and `...Finished` methods, but it felt way too spammy, so I settled for the messages in `SyncFinished` and `maybeDecreaseBuffer`, which prints something whenever the `syncBufferCap` changes.  And then `Shutdown` makes sure the byte counts are always 0 at the end of each test.

I'm happy to add more logs if you think it would be helpful, I'm just not sure where exactly you're thinking about.

(I've also thought about adding some of these numbers to the top-level `.kbfs_status` file, but I'm reluctant to cause the `kbfs-server` breakage.  That's a bad excuse though.)